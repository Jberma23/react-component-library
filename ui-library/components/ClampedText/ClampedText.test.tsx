import { render } from "@testing-library/react";
import { ClampedText } from ".";

const testString = "short king test caption";
describe(ClampedText, () => {
  it("renders with required props", () => {
    const { asFragment } = render(<ClampedText>{testString}</ClampedText>);
    expect(asFragment()).toMatchSnapshot();
  });

  it("passes all a11y checks", () => {
    expect(<ClampedText>{testString}</ClampedText>).toBeA11yWhenRendered();
  });
});
