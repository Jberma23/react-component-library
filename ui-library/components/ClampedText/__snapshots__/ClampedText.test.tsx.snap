// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClampedText renders with required props 1`] = `
<DocumentFragment>
  <div>
    <div
      class="text-body-regular line-clamp-3"
    >
      short king test caption
    </div>
  </div>
</DocumentFragment>
`;
